code/acting/src/acting/pure_pursuit_controller.py

```diff
@@ -161,13 +163,16 @@ def __calculate_steer(self) -> float:
         :return:
         """
         l_vehicle = 2.85  # wheelbase
-        k_ld = 2.0  # todo: tune
-        look_ahead_dist = 5.0  # offset so that ld is never zero
-
-        if round(self.__velocity, 1) < 0.1:
-            look_ahead_dist += 1.0
+        k_ld = 1.0  # todo: tune
+        look_ahead_dist = 3.5  # offset so that ld is never zero
+
+        if self.__velocity < 0:
+            # backwards driving is not supported
+            return 0.0
+        elif round(self.__velocity, 1) < MIN_LD_V:
+            look_ahead_dist += 0.0
```

Comment:
`+=` doesn't make much sense. Do you mean `=`?
```suggestion
            look_ahead_dist = 0.0
```

Reply:
`+=` is the intended operator. `look_ahead_dist` is declared with a certain value (atm 3.5) this is the minimum ld dist. When the vehicle is moving slowly e.g. during a parking maneuver an offset is added to this distance (atm += 0, so no offset). When the vehicle is moving faster than `MIN_LD_V` a speed dependent factor is added. I changed this implementation in order to avoid jumps in the ld dist. In previous version this value could jump, which might result in oscillations in the steering commands.

Reply:
Ok, it just looks weird because the line doesn't do anything

Reply:
I agree. I had to tune the ld dist quite a bit and sometimes I added an offset here. I will add a comment to avoid confussion.

Reply:
fixed in [d556b8e](https://github.com/ll7/paf22/pull/210/commits/d556b8ea0a8a784c0dcf05d11e1a526768c9dfce)